### DISCUSSION TITLE
Adding a theme

### DISCUSSION THREAD

**Discussion by plasmoduck** [LazyVim/LazyVim#4354] [Q&A]
How can I correctly add [this](https://github.com/samueljoli/cyberpunk.nvim) nvim theme?

 Bellow is a screen shot of how I tried to enable it in my ~/.config/nvim/lua/plugins/colorscheme.lua and how it looks.

![2024-08-24_23-40-23](https://github.com/user-attachments/assets/0bd41d36-77a6-444b-bb02-9fe72de12126)

This is how it is supposed to look.

![cyberpunk](https://github.com/user-attachments/assets/c8c9bfa0-ae97-47bb-a33d-7bffb10a79b2)

**Comment by rogeriofrsouza** [comment]
Seems like the code is fine.
Did you install the plugin?
